ui/constants/stripe.js

```diff
@@ -1,2 +1,23 @@
+import { URL, WEBPACK_WEB_PORT } from 'config';
+import * as ICONS from 'constants/icons';
+
 export const TEST = 'test';
 export const LIVE = 'live';
+
+const isDev = process.env.NODE_ENV !== 'production';
+
+export const SUCCESS_ENDPOINT = '/$/settings/tip_account';
+export const FAILURE_ENDPOINT = '/$/settings/tip_account';
+
+const stripeRedirectPath = isDev ? 'http://localhost:' + WEBPACK_WEB_PORT : URL;
+export const SUCCESS_REDIRECT_URL = stripeRedirectPath + SUCCESS_ENDPOINT;
+export const FAILURE_REDIRECT_URL = stripeRedirectPath + FAILURE_ENDPOINT;
+
+export const APIS_DOWN_ERROR_RESPONSE = __('There was an error from the server, please try again later');
+export const CARD_SETUP_ERROR_RESPONSE = __('There was an error getting your card setup, please try again later');
```

Comment:
:small_red_triangle: Don't run translation macro at file/global scope level

These only run once per refresh, so the translation is locked to whatever language it was at during eval.
Apply the translation macro is a render function -- that way, it can respond to language state changes.